### ISSUE TITLE
IBM ID badge rebrand

### ISSUE THREAD

**Issue by mitrousis** [ibm-brand/center-import-test#21]
_kenhartmann created the following on Feb 27:_  

 Waiting on the IBM Security team to deliver prints.

Pete's original studies captured here: https://zenhub.ibm.com/app/workspaces/-bsxd-master-workspace-5d5fd4b90c9aa55544db8833/issues/brand/design-language/4 

_Original issue: https://github.ibm.com/brand/center/issues/750_

**Comment by mitrousis** [comment]
_On May 11, mjabbink commented:_
@rw-us and @llind Any words on tests from Marc Wisniewski?

**Comment by mitrousis** [comment]
_On Jun 10, mjabbink commented:_
@llind Did we ever hear back on this?

**Comment by mitrousis** [comment]
_On Jul 01, mjabbink commented:_
@llind and @kenhartmann Would be great to get an update or status from CIO team. I believe it was Marc Wisniewski

**Comment by mitrousis** [comment]
_On Jul 02, kenhartmann commented:_
Hi @mjabbink,

From Marc...

"That work was put on the back-burner for a bit until we’re capable of getting back into the office.

Some more context:
![image (6).png](https://zenhub.ibm.com/images/585016c4fdfd5b9223bbf9f4/c93c9b1c-e8d1-4590-9d82-37e6ce9e8deb)



Note the visual defects in the above comps that were prepared for us from the vendor in the UK that produces the software we use to produce our badges.

The net of this is that we won’t be able to move this forward again until we are back in the office and can test on the printers we have in house.

Hope that helps!"

**Comment by mitrousis** [comment]
_On Jul 02, mjabbink commented:_
Thanks @kenhartmann Is there a way to include Marc for this repo so we can stay coordinated in this effort. The brand is a quintessential piece for the IBMer and our team. Happy to work together with Marc’s team on this and @Peter-Garvin has already done so by sharing his badge layouts and suggestions. We just need to continue the collaboration as one team. Not CIO and BXD but ONE team.

**Comment by mitrousis** [comment]
_On Jul 02, kenhartmann commented:_
> Thanks @kenhartmann Is there a way to include Marc for this repo so we can stay coordinated in this effort. The brand is a quintessential piece for the IBMer and our team. Happy to work together with Marc’s team on this and @Peter-Garvin has already done so by sharing his badge layouts and suggestions. We just need to continue the collaboration as one team. Not CIO and BXD but ONE team.

Good idea @mjabbink. @marcwis Welcome to the Brand Center repo! It would be great to keep work effort updates on this work stream here–sound good?

**Comment by mitrousis** [comment]
_On Jul 02, marcwis commented:_
If it's easier I can invite you all to a Slack channel with Chris (the stakeholder for this work).  We previously communicated to both Chris and Russ where we were on this and that we were paused until we're back in the office.

**Comment by mitrousis** [comment]
_On Jul 02, marcwis commented:_
Related, attaching clearer images from our internal channel on this.
![IBM1](https://media.github.ibm.com/user/261/files/b401ab00-bc82-11ea-8fec-07ae53d5e4b8)
![IBM2](https://media.github.ibm.com/user/261/files/b401ab00-bc82-11ea-86e3-ff49247e0e6c)

**Comment by mitrousis** [comment]
_On Jul 02, kenhartmann commented:_
> If it's easier I can invite you all to a Slack channel with Chris (the stakeholder for this work). We previously communicated to both Chris and Russ where we were on this and that we were paused until we're back in the office.

Amazing, yes, please add @llind and @mjabbink to the slack channel–thank you!

**Comment by mitrousis** [comment]
_On Jul 02, mjabbink commented:_
@marcwis yes, good idea. Terry Yoo is also a stack holder hence our need to collaborate. Tests seem promising and ideally we could get physical samples sent to us. I’d love to see the them and suggest we get tests done from across several printers and goes to see the variations in quality and color. 

I lean on the designs that do not have the large image because our images are going to be lame and no need for them to be so large.

Also that is not the **think** logotype in that sample.

**Comment by mitrousis** [comment]
_On Jul 06, marcwis commented:_
https://cio-design.slack.com/archives/G016LSP282W is the new shared channel.

**Comment by mitrousis** [comment]
_On Aug 24, sadek commented:_
@Peter-Garvin have you been tracking on any of this by chance? In the above Slack channel or otherwise?

**Comment by mitrousis** [comment]
_On Aug 24, marcwis commented:_
FYI... We are still in a holding pattern on this as we need someone from Security (likely still Ed Collins) to produce additional test badges. I've pinged him again in the shared channel for status.

**Comment by mitrousis** [comment]
_On Aug 24, sadek commented:_
Thanks for the update @marcwis I'm trying to do some seasonal grooming on our outstanding issues so this helps.

**Comment by mitrousis** [comment]
_On Sep 01, mjabbink commented:_
These are nice.

<img width="160" alt="Screen Shot 2020-07-07 at 11 05 52 AM" src="https://media.github.ibm.com/user/33519/files/40592b80-ec3c-11ea-91be-8347fb899642">


———————


<img width="166" alt="Screen Shot 2020-07-07 at 11 05 57 AM" src="https://media.github.ibm.com/user/33519/files/4a7b2a00-ec3c-11ea-9d9e-455b717f7dd6">

**Comment by mitrousis** [comment]
_On Sep 01, mjabbink commented:_
— Logos should be same size between the two
— Whrer the type starts (bottom baseline) should be the same between the two
